docroot/sites/all/modules/features/forms/forms.module

```diff
@@ -550,12 +602,24 @@ function _year_as_list_text_options($field, $instance = NULL, $entity_type = NUL
  * I need this, because of forms that have more than one field for a set of years
  */
 function _forms_year_as_list_text_options2($field, $instance = NULL, $entity_type = NULL, $entity = NULL) {
-  if ($entity->type == 'contact_ilr_admissions') {
-    // Anticipated entry year at Cornell for Admissions Contact form
-    // If it's before April: start with current year and go out 6 years
-    // April or later: start with the following year and go out 6 years
-    return date('n') < 4 ? _forms_year_relative_options(0,6) : _forms_year_relative_options(1,6);
-  }
+
+    if ($entity->type == 'contact_ilr_admissions') {
```

Comment:
Please don't use both if/else and switch statements. Go ahead and put them all in the switch.